Source/WarpX.H

```diff
@@ -708,6 +720,12 @@ private:
      * `load_balance_knapsack_factor=2` limits the maximum number of boxes that can
      * be assigned to a rank to 8. */
     amrex::Real load_balance_knapsack_factor = 1.24;
+    /** Threshold value that controls whether to adopt the proposed distribution
+     * mapping during load balancing.  The new distribution mapping is adopted
+     * if the ratio of proposed distribution mapping efficiency to current
+     * distribution mapping efficiency is larger than the threshold; 'efficiency'
+     * here means the average cost per MPI rank.  */
+    amrex::Real load_balance_efficiency_ratio_threshold = 0;
```

Comment:
I know this is arbitrary, but ideally the default option would start a good enough simulation.
```suggestion
    amrex::Real load_balance_efficiency_ratio_threshold = 1.1;
```